oak/server/rust/oak_runtime/src/runtime/tests.rs

```diff
@@ -106,38 +190,72 @@ fn create_node_invalid_configuration() {
     );
 }
 
-/// Create a test Node that creates a Node with a more public label and fails.
+/// Create a test Node that creates a Node with a less secret label and fails.
 ///
 /// If this succeeded, it would be a violation of information flow control, since the original
-/// secret Node would be able to spawn "public" Nodes and use their side effects as a covert
-/// channel to exfiltrate secret data.
+/// secret Node would be able to spawn "less secret / public" Nodes and use their side effects as a
+/// covert channel to exfiltrate secret data.
 #[test]
-fn create_node_more_public_label() {
-    let secret_label = Label {
+fn create_node_less_secret_label_err() {
+    let initial_label = Label {
         secrecy_tags: vec![oak_abi::label::authorization_bearer_token_hmac_tag(&[
             1, 1, 1,
         ])],
         integrity_tags: vec![],
     };
+    let less_secret_label = Label {
+        secrecy_tags: vec![],
```

Comment:
Any reason for not using `public_trusted`?

Reply:
Same as the reason above :) I want to be able to see the difference between the two spelled out.